src/ci/docker-compose/docker-compose-horicromtal.yml

```diff
@@ -79,6 +92,8 @@ services:
         -Dwebservice.port=${MANAGED_CROMWELL_PORT-8008}
         -Dsystem.cromwell_id=frontend
         -Dservices.MetadataService.config.metadata-summary-refresh-interval=Inf
+        -Dservices.MetadataService.config.carbonite-metadata-service.enabled=false
```

Comment:
TOL Not an issue with your PR and I'm sure I've said this before, but `enabled` here is kind of confusing. The frontend does "enable" Carboniting regardless of setting `carbonite-metadata-service.enabled = false` since it will look in the `carbonite-metadata-service.bucket` to find Carbonited workflow JSONs .